.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,54 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches: "master"
+    paths-ignore: ['**.md']
+  push:
+    branches: "master"
+    paths-ignore: ['**.md']
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  generate-lockfile:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: cargo generate-lockfile -v
```

Comment:
Yes it will, but we don't publish a lockfile within this repo. Are you saying that we should or that there's some other way to use lockfiles here that I'm not aware of.

I thought we decided not to publish lockfiles for our library modules.